### DISCUSSION TITLE
Dropping Objective-C Support

### DISCUSSION THREAD

**Discussion by scannillo** [braintree/braintree_ios#1431] [Polls]
Hello folks. Thanks for using the Braintree iOS SDK and for all of your valuable feedback. As we plan for the future major versions of this SDK we want to understand how you all integrate.

We are considering dropping Objective-C support in the future so we can focus on modernizing and evolving alongside the Swift language more quickly.

Please respond to the poll below with how this would impact your app. Any comments/ideas/concerns always welcome!

**Comment by jmpdi** [comment]
I recommend that Braintree maintain Objective-C support for better interoperability with hybrid and cross-platform development environments.

Also, thank you Braintree for opening this poll and allowing developers to have their voices heard before making such a major decision about the SDK.  👍
